### PR TITLE
Renamed node 'disconnect' function overrides to instead override 'detach'

### PR DESCRIPTION
**Pull request by JamesOrdner**
This updates nodes which had not been changed to reflect the depreciation of disconnect(). This will allow easier node cleanup for all nodes with a single node.detach() call as designed.

### CODE REVIEW

**Comment by aure** [comment]
Sorry I hadn't seen this pull request till now.
